tests/test_copy.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+import os
 import filecmp
```

Comment:
Ruff is unhappy because of the import order:

```suggestion
import filecmp
import os
```